examples/GetNPUsers.py

```diff
@@ -318,9 +318,11 @@ def run(self):
     def request_users_file_TGTs(self):
 
         with open(self.__usersFile) as fi:
-            usernames = [line.strip() for line in fi]
+          def _usernames(fi):
+            for line in fi:
+              yield line
```

Comment:
Please preserve the `.strip()` call from the original code to avoid trailing `\n` newlines at the end of each line,
i.e.: `yield line.strip()`.